testng-core/src/test/kotlin/org/testng/util/StringsTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.testng.util
+
+import org.assertj.core.api.Assertions.assertThat
+import org.testng.annotations.Test
+
+class StringsTest {
+    @Test
+    fun joinEmptyArray() {
+        val emptyArray = arrayOf<String>()
+        assertThat(Strings.join(",", emptyArray)).isEmpty()
+    }
+
+    @Test
+    fun joinArrayWithOneElement() {
+        val array = arrayOf("one")
+        assertThat(Strings.join(",", array)).isEqualTo("one")
```

Comment:
It is not critical here but WDYT about the "Given, When, Then" pattern on tests?

ie:

```
val array = arrayOf("one")
val actual = Strings.join(",", array)
assertThat(actual).isEqualTo("one")
```